### PR TITLE
Declare offering delivery method and unit.

### PR DESCRIPTION
**Pull request by igwejk**
This change should facilitate implementation of automations that consumes the offering feed e.g. scoping enhancement https://github.com/github/services/pull/19929

### CODE REVIEW

**Review by umatty — APPROVED** [review]

**Comment by stoe** [comment]
I converted to _draft_ for now, to not send wrong signal before we have discussed and confirmed that the proposed change will not break `github/github`

File: _offerings/offering-ghas-security-advisory-services.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 ---
 layout: page
 title: GitHub Advanced Security - Security Advisory Services
+delivery:
+  method: "A minimum committed hours of 4 hours per week; committed hours may be more based on contract duration."
```
Comment (stoe) [inline _offerings/offering-ghas-security-advisory-services.md:5]:
```suggestion
  method: "A minimum commitment of 4 hours per week; committed hours may be more based on contract duration."
```

File: _offerings/offering-technical-advisory-services.md [hunk 2]
```diff
@@ -1,6 +1,9 @@
 ---
 layout: page
 title: GitHub Technical Advisory Services
+delivery:
+  method: "A minimum committed hours of 4 hours per week; committed hours may be more based on contract duration."
```
Comment (stoe) [inline _offerings/offering-technical-advisory-services.md:5]:
```suggestion
  method: "A minimum commitment of 4 hours per week; committed hours may be more based on contract duration."
```

**Review by stoe — CHANGES_REQUESTED** [review]

**Comment by stoe** [comment]
@igwejk, do think we could add a test to https://github.com/github/github/blob/master/test/lib/explore_feed/professional_service_test.rb to ensure the additional fields get ignored?

**Review by stoe — COMMENTED** [review]
Visual nit-pick on the rendered MD...
I think the `delivery` section should be at the end of the rendered MD, after `parameterized_name`.

But maybe that's just me ¯\_(ツ)_/¯

image

**Review by stoe — APPROVED** [review]

**Pull request merged**